ui/app/actions.js

```diff
@@ -277,10 +277,10 @@ function signMsg (msgData) {
 }
 
 function signTx (txData) {
+  _accountManager.setGasMultiplier(txData.gasMultiplier)
```

Comment:
It's a little strange to call to the `accountManager` without a callback. How do you know the `accountManager` is done setting the `gasMultiplier` before you `sendTransaction`?